Jucer2Reprojucer/CMakeLists.txt

```diff
@@ -103,7 +103,7 @@ jucer_project_module(
 
 jucer_export_target(
   "Xcode (MacOSX)"
-  COMPILER_FLAGS_FOR_maximum_warnings "-Werror -Weverything -Wno-c++98-compat -Wno-c++98-compat-pedantic -Wno-padded"
+  COMPILER_FLAGS_FOR_maximum_warnings "-Werror -Weverything -Wno-c++98-compat -Wno-c++98-compat-pedantic -Wno-exit-time-destructors -Wno-padded"
```

Comment:
Adding `-Wno-exit-time-destructors` to the compiler flags of `Jucer2Reprojucer` should be done in the same commit that introduces the static variables, otherwise that commit won't build.